docs/ops.html

```diff
@@ -1279,8 +1329,13 @@
         <th>Mbean name</th>
       </tr>
       <tr>
-        <td>commit-latency-[avg | max]</td>
-        <td>The [average | maximum] commit time in ns for this task. </td>
+        <td>commit-latency-avg</td>
+        <td>The average commit time in ns for this task. </td>
+        <td>kafka.streams:type=stream-task-metrics,streams-task-id=([-.\w]+)</td>
```

Comment:
It is incorrect: from I saw it is `kafka.streams:type=stream-task-metrics,client-id=[thread-name],streams-task-id=[task-id]`.

Ditto for processor-node-level, and store-level metrics.